README.md

```diff
@@ -97,6 +97,14 @@ tns plugin add nativescript-twilio
     },
     onPushRegisterFailure (error) {
       dialogs.alert(`push registration failed: ${error}`);
+    },
+
```

Comment:
Nit: please remove this empty line in order to have a common spacing

Reply:
Thanks Manuel. Fixed this